packages/frontend/src/features/scheduler/components/SchedulerFilters.tsx

```diff
@@ -24,19 +26,29 @@ import {
 import { useProject } from '../../../hooks/useProject';
 import { useDashboardContext } from '../../../providers/DashboardProvider';
 
+type SchedulerFilterRule = DashboardFilterRule & {
+    tileTargets: undefined;
```

Comment:
do we want it `undefined` or optional? 
maybe... `optional`? so we don't need to add this line. What do you think?

Reply:
@joaoviana good question! ok so I was thinking about this a lot. whenever you update filter it returns new `DashboardFilterRule` and it includes `tileTargets`.

typescript does not complain whenever you have additional properties in the object so it is really easy to forget removing that property and thought to make it explicit - `SchedulerFilterRule` should  contain `tileTargets` equal to undefined.

let me know if this is useless and we should take care of it on the BE side?

Reply:
ahhhhh! I see!! Amazing, let's keep your change! 👍